pkg/cluster/member.go

```diff
@@ -64,3 +71,12 @@ func podsToMemberSet(pods []*api.Pod, selfHosted *spec.SelfHostedPolicy) etcduti
 	}
 	return members
 }
+
+func findID(name string) (int, error) {
```

Comment:
The logic of naming convention relies in pkg/cluster:

https://github.com/coreos/etcd-operator/blob/master/pkg/cluster/cluster.go#L297

Can we put it in the same pkg right now? I have another PR ontop.
We would centralize such logic, probably in etcdutil/member later.